scaleway/helpers_vpc.go

```diff
@@ -51,13 +51,13 @@ func vpcAPIWithRegionAndID(m interface{}, id string) (*v2.API, scw.Region, strin
 	return vpcAPI, region, ID, err
 }
 
-func vpcAPI(m interface{}) (*v1.API, error) {
+func vpcAPI(m interface{}) (*v2.API, error) {
```

Comment:
Can the import on vpc v1 be removed? That way we have a full consistant and vpc v2 api usage?